Scala/Scala.sublime-syntax

```diff
@@ -1402,17 +1402,17 @@ contexts:
       scope: punctuation.definition.string.begin.scala
       push:
         - meta_include_prototype: false
-        - meta_scope: string.quoted.triple.scala
-        - match: '{{unicode_char}}'
-          scope: constant.character.escape.scala
+        - meta_scope: meta.string.scala string.quoted.double.block.scala
```

Comment:
Triple strings are different from double (similar to Bash).

Reply:
The comparison here is to Python's `"""`.

Reply:
Bash supporting this style of strings?

Took those scopes from Python, which supports both "triple-double" (`"""`) and triple-single (`'''`) quotes and thus wants to distinguish them.

Just see C# and Java also use `quoted.triple` as they only support `"""`. Thus we could probably follow them as well.